babel.server.js

```diff
@@ -1,5 +1,12 @@
 require('babel-register');
 
+var cssModulesHook = require('css-modules-require-hook');
+
+cssModulesHook({
+  generateScopedName: '[name]__[local]___[hash:base64:5]',
```

Comment:
Ideally, you want this in development, but a shorthash in production for better minification.